### PR TITLE
added v8 and prepared v9

### PR DESCRIPTION
**Pull request by dominikwilkowski**

### CODE REVIEW

File: content/v8.0.0/index.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+layout: event
+
+header: /_shared/header.md
+
+type: event
+pagetitle: Design System meetup Sydney v8.0.0
+version: v8.0.0
+city: Sydney
+date: 2019-05-13
+location: /_locations/hipages.md
```
Comment (alex-e-leon) [inline content/v8.0.0/index.yml]: location should be canva

File: content/v8.0.0/index.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+sponsors:
+  - /_sponsors/canva.md
+  - /_sponsors/thinkmill.md
+  - /_sponsors/tomwalker.md
+link: https://www.meetup.com/Design-System/events/258516904
+eventdescription: |
+  Design System meetup Sydney v8.0.0 is up.
```
Comment (alex-e-leon) [inline content/v8.0.0/index.yml:16]: minor - but shouldn't the event description for events that have already passed just be `Design System meetup Sydney v8.0.0`? is up seems strange for events that are already past.

LMK if you agree, and I can separately fix it up for all events in another PR.

**Review by alex-e-leon — COMMENTED** [review]
LGTM - only 1 minor comment and a location fix - otherwise looks good though.

**Review by alex-e-leon — APPROVED** [review]

File: content/v8.0.0/index.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+pagetitle: Design System meetup Sydney v8.0.0
+version: v8.0.0
+city: Sydney
+date: 2019-05-13
+location: /_locations/hipages.md
+sponsors:
+  - /_sponsors/canva.md
+  - /_sponsors/thinkmill.md
+  - /_sponsors/tomwalker.md
+link: https://www.meetup.com/Design-System/events/258516904
```
Comment (alex-e-leon) [inline content/v8.0.0/index.yml]: Just caught - this is the url for 7.0.0 meetup - needs to be updated too.

File: content/v8.0.0/index.yml (hunk 1 above)
Comment (dominikwilkowski) [reply content/v8.0.0/index.yml]: fixed

File: content/v8.0.0/index.yml (hunk 2 above)
Comment (dominikwilkowski) [reply content/v8.0.0/index.yml:16]: meh... it's for better infos later... just haven't come up with them

File: content/v8.0.0/index.yml (hunk 3 above)
Comment (dominikwilkowski) [reply content/v8.0.0/index.yml]: fixed

**Pull request merged**